frontend/src/components/core/StreamlitDialog/StreamlitDialog.test.tsx

```diff
@@ -73,3 +75,53 @@ describe("StreamlitDialog", () => {
     expect(wrapper.find("StyledTertiaryButton")).toMatchSnapshot()
   })
 })
+
+describe("aboutDialog", () => {
+  beforeEach(() => {
+    SessionInfo.current = new SessionInfo({
+      appId: "aid",
+      sessionId: "sessionId",
+      streamlitVersion: "42.42.42",
+      pythonVersion: "pv",
+      installationId: "iid",
+      installationIdV3: "iid3",
+      authorEmail: "ae",
+      maxCachedMessageAge: 2,
+      commandLine: "command line",
+      userMapboxToken: "mpt",
+    } as SessionInfoArgs)
+  })
+
+  afterEach(() => {
+    const UnsafeSessionInfo = SessionInfo as any
+    UnsafeSessionInfo.singleton = undefined
+  })
+
+  it("renders about dialog with Streamlit version included", async () => {
+    const wrapper = mount(
+      <Fragment>
+        {StreamlitDialog({
+          type: DialogType.ABOUT,
+          onClose: () => {},
+        })}
+      </Fragment>
+    )
+
+    expect(wrapper.find(Modal).text()).toContain("Streamlit v42.42.42")
+  })
+
+  it("renders about dialog without Streamlit version included", async () => {
+    SessionInfo.clearSession()
+
+    const wrapper = mount(
+      <Fragment>
+        {StreamlitDialog({
+          type: DialogType.ABOUT,
+          onClose: () => {},
+        })}
+      </Fragment>
+    )
+    expect(wrapper.find(Modal).exists()).toBe(true)
+    expect(wrapper.find(Modal).text()).not.toContain("Streamlit v42.42.42")
```

Comment:
Nit: this should probably be `expect(wrapper.find(Modal).text()).not.toContain("Streamlit v")` - we are testing that _no_ version string is displayed